docs/examples/resnet50_trt/model_repository/resnet50_trt/config.pbtxt

```diff
@@ -26,7 +26,7 @@ input [
 {
     name: "input"
     data_type: TYPE_FP32
-    dims: [ 3, -1, -1 ]
+    dims: [ 3, 224, 224 ]
```

Comment:
Is there a reason why it now needs to be specified explicitly? Is it worth adding a comment?